_drafts/oryx/browser-compatibility.md

```diff
@@ -0,0 +1,24 @@
+# Supported browsers / viewports
+Oryx supports and is being tested on all modern browsers and operating systems:
+
+* iOS 16.x (Safari latest, Google Chrome latest)
+* Android 13.x (Google Chrome latest)
+* macOS Ventura (Safari latest, Google Chrome latest) 
+* Windows 10 and higher (Google Chrome latest, Edge latest)
+
+This allows us to cover more than 95% of operating systems and 93% of browsers used worldwide.
+
+---
+
+As for screen resolutions, we test our components on real devices: 
```

Comment:
But we are testing not only components but the whole frontend, built with these components and I think it's even more important for customers - that's the whole product is tested and working.

Reply:
Also removed that `testing` info as it might be not relevant for this file at all